css/style.css

```diff
@@ -0,0 +1,551 @@
+    position: relative;
+}
+
+.location-list{
+    float: left;
+    position: absolute;
+    top: 220px;
+}
+
+.location-one img{
+    height: 45px;
+    width: 45px;
+}
+.location li{
+    display: flex;
+    margin-top: 10px;
+}
+.location li span{
+    margin-top: 10px;
+    color: #ffffff;;
+}
+.about-plan{
+    /* position: absolute; */
+    /* left: 100px; */
+    width: 85%;
+    background-color: rgb(51, 87, 153);
+    margin: 10px 0;
+    padding-bottom: 10px;
+    margin-left: 50px;
+}
+.about-plan p{
+    text-decoration: underline;
+    color: rgb(238, 184, 67);
+    font-size: 16px;
+    width: 100%;
+    margin-left: 30px;
+}
+
+.about-plan pre{
+    font-size: 12px;
+    color: #ffffff;
+    margin-left: 30px;
+}
+
+.location-list li::after{
+    content: '';
+    height: 8%;
+    width: 1px;
+    position: absolute;
+    left: 45px;
+    margin-top: 40px;
+    background-color: #ffffff;
+}
+.location-list li::after:nth-child(4){
+    width: 0px;
+    height: 0px;
+}
+
+/* main-content starts */
+.plan-right-content{
+    width: 75%;
+    float: left;
+    margin-top: 20px;
+}
+
+.product-details li{
+    float: left;
+    padding: 20px 63px;
+    /* border-right: 1px solid #3c3c3d;
+    color: rgb(51, 50, 50); */
+    position: relative;
+}
+.product-details li.product-info::after{
+    content:' ';
+    height: 50%;
+    width: 2px;
+    position: absolute;
+    top: 25px;
+    right: 0px;
+    background-color: rgb(211, 209, 209);
+}
+.about-product{
+    float: left;
+    width: 96%;
+    background-color: #ffffff;
+    text-align: center;
+    margin-left: 20px;
+    clear: both;
+    box-shadow: 1px 2px #b7b4b4;
+}
+.product-info span{
+    font-size: 22px;
+    font-weight: 600;
+}
+.product-info-alert span{
+    font-size: 22px;
+    font-weight: 600;
+}
+
+.active-partner{
+    float: left;
+    width: 96%;
+    margin-left: 20px;
+}
+.active-partner-count{
+    padding: 18px 0;
+    float: left;
+}
+.active-partner-count span{
+    color: #292828;
+}
+.active-partner-count span.inactive{
+    font-size: 13px;
+    color: #6e6d6d;
+}
+.buttons{
+    display: inline-flex;
+    float: right;
+}
+.date{
+    width: 50%;
+}
+.filter{
+    width: 50%
+}
+.plans-date{
+    width: 90%;
+    padding: 6px 30px;
+    margin:10px 10px 0px 15px;
+    background-color: #505fe6;
+    outline: none;
+    border: none;
+    color: #ffffff;
+    font-size: 14px;
+    font-weight: 200;
+}
+
+.plans-detail{
+    width: 90%;
+    padding: 6px 30px;
+    margin:10px 0px 0px 15px;
+    background-color: #38ad86;
+    outline: none;
+    border: none;
+    color: #ffffff;
+    font-size: 14px;
+    font-weight: 200;
+}
+
+.compare-partner{
+    float: left;
+    width: 96%;
+    background-color: #ffffff;
+    text-align: center;
+    margin-left: 20px;
+    clear: both;
+    padding: 12px 0;
+}
+
+.compare-partner span{
+    color: #4b4a4a;
+}
+.compare-partner span.compare-button{
+   padding: 6px 15px;
+   background-color: darkorange;
```

Comment:
@lakshmikhabbas Always use hex color code. Writing color names is not a good practice.

Reply:
okay